<ui version="4.0" >
 <class>fm_viewSchema</class>
 <widget class="QWidget" name="fm_viewSchema" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Schema View</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QSlider" name="vs_zoom" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QGraphicsView" name="gv_schema" />
       </item>
       <item>
        <widget class="QSlider" name="hs_position" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QSlider" name="vs_position" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDial" name="dial" />
     </item>
     <item>
      <widget class="QPushButton" name="pb_viewReset" >
       <property name="text" >
        <string>reset view</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
